cmds/ls/fileinfo_linux.go

```diff
@@ -125,4 +137,9 @@ func (fi longStringer) String() string {
 		fi.size,
 		fi.modTime.Format("Jan _2 15:04"),
 		fi.comp.String())
+
+	if fi.mode&os.ModeType == os.ModeSymlink {
+		s += fmt.Sprintf(" -> %v", fi.symlink)
```

Comment:
I'm not sure what the best practice is for printing out filenames unescaped, but in theory couldn't it contain all sorts of ansi escape sequences?